back/engines/free/email_campaigns/app/models/email_campaigns/campaigns/admin_digest.rb

```diff
@@ -108,30 +108,20 @@ def user_filter_no_invitees(users_scope, _options = {})
 
     def content_worth_sending?(_)
       [
-        statistics.dig(:activities, :new_ideas, :increase),
-        statistics.dig(:activities, :new_initiatives, :increase),
-        statistics.dig(:activities, :new_comments, :increase),
-        statistics.dig(:users, :new_visitors, :increase),
-        statistics.dig(:users, :new_users, :increase),
-        statistics.dig(:users, :active_users, :increase)
+        statistics.dig(:activities, :new_ideas_count),
+        statistics.dig(:activities, :new_comments_count),
+        statistics.dig(:users, :new_users_count)
       ].any?(&:positive?)
     end
 
     def statistics
       @statistics ||= {
         activities: {
-          new_ideas: stat_increase(Idea.pluck(:published_at).compact),
-          new_initiatives: stat_increase(Initiative.pluck(:published_at).compact),
-          new_reactions: stat_increase(Reaction.pluck(:created_at)),
```

Comment:
Are we no longer interested in proposals and reactions as an indicator of activity?

Reply:
Or is what you are saying that these stats were never actually called in the emails anyway. Which is what it looks like below.

Reply:
Yes, they are/were never used in the mails.